librz/asm/arch/rl78/rl78.c

```diff
@@ -35,7 +35,38 @@ bool rl78_dis(RL78Instr RZ_OUT *instr, size_t RZ_OUT *bytes_read,
                 byte = buf[next_byte_p++];
         }
 
-        int map;
+        // special cases
+        if (next_byte_p + 1 < buf_len) {
+                // MULHU, MULH...
+                if (byte == 0xCE && buf[next_byte_p] == 0xFB) {
```

Comment:
If those magic numbers (`0xCE`...) can be expressed with a name, please `#define` them somewhere.